src/include/btmem.h

```diff
@@ -351,6 +351,8 @@ struct __wt_page_modify {
                 WT_ADDR addr;
                 uint32_t size;
                 uint32_t checksum;
+
+                bool restore;
```

Comment:
Since `multi.restore` only applies to entries on the saved-update list, I would suggest a name that reflects that (maybe `supd_restore`?), and also moving it up immediately after the `supd_entries` field. That also drops the size of the WT_MULTI structure by 8B, no idea if that makes a difference though.

Reply:
changed as suggested.